test/e2e/components/vl-datepicker.js

```diff
@@ -0,0 +1,252 @@
+        }
+        return (await this._getToggleButton()).click();
+    }
+
+    async _getDays() {
+        return (await this._getWrapper()).findElements(By.css('span.flatpickr-day'));
+    }
+
+    async _getDaysMap() {
+        const allDays = await this._getDays();
+        return Promise.all(allDays.map(async (day) => {
+            const text = await day.getText();
+            const isPreviousMonth = (await day.getAttribute('class')).includes('prevMonthDay');
+            return { text: text, webElement: day, isPreviousMonth: isPreviousMonth }
+        }));
+    }
+
+    async _getHours() {
+        const input = await (await this._getWrapper()).findElement(By.css('input.flatpickr-hour'));
+        return input.getAttribute('value');
+    }
+
+    async _getMinutes() {
+        const input = await (await this._getWrapper()).findElement(By.css('input.flatpickr-minute'));
+        return input.getAttribute('value');
+    }
+
+    async _increase(ticker) {
+        const tickerWrapper = await ticker.findElement(By.xpath('..'));
+        const arrowUp = await tickerWrapper.findElement(By.css('span.arrowUp'));
+        return arrowUp.click();
+    }
+
+    async takeScreenshot(driver, file){
+        let image = await driver.takeScreenshot();
+        await writeFile(file, image, 'base64');
+      }
+
+    async _increaseWith(ticker, times) {
+        for (let index = 0; index < times; index++) {
+            await this._increase(ticker);
+        }
+        return Promise.resolve();
+    }
+
+    async _decreaseWith(ticker, times) {
+        for (let index = 0; index < times; index++) {
+            await this._decrease(ticker);
+        }
+        return Promise.resolve();
+    }
+
+    async _decrease(ticker) {
+        const tickerWrapper = await ticker.findElement(By.xpath('..'));
+        const arrowDown = await tickerWrapper.findElement(By.css('span.arrowDown'));
+        return arrowDown.click();
+    }
+
+    async _getSelectedYear() {
+        const input = await (await this._getWrapper()).findElement(By.css('.numInput'));
+        return input.getAttribute('value');
+    }
+
+    async _calculateDifference(value, originalValue, minutes) {
+        let difference = value - originalValue;
+        if (minutes) {
+            difference = Math.floor(difference / 5);
+        }
+        return Math.abs(difference);
+    }
+
+    async _setValueInTicker(ticker, minutes, value) {
+        await this._openFlatpickr();
+        const originalValue = await (await ticker).getAttribute('value');
+        if (value > originalValue) {
+            const difference = await this._calculateDifference(value, originalValue, minutes);
+            await this._increaseWith(ticker, difference);
+        } else {
+            const difference = await this._calculateDifference(value, originalValue, minutes);
+            await this._decreaseWith(ticker, difference);
+        }
+        return Promise.resolve();
+    }
```

Comment:
ook geen manier nodig om uren + minuten via een input veld te zetten? of heeft de component dat niet?